web/views.py

```
@@ -7599,3 +7601,56 @@ def add_contributor(username, avatar_url, profile_url):
+        "requests": requests,
+        "categories": Subject.objects.all(),
+        "category_counts": category_counts,
+        "selected_category": selected_category,
+        "selected_category_display": selected_category_display,
+    }
+
+    return render(request, "videos/request_list.html", context)
+
+
+@login_required
+def submit_video_request(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `submit_video_request`

(ANN201)